### PR TITLE
feat: build backend assets with docker

### PR DESCRIPTION
**Pull request by fjogeleit**
* Build frontend within docker and copy it to the backend

### CODE REVIEW

File: scripts/docker-build-entrypoint.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+cd /var/www/
```
[the remark this answers is not in the document]
Comment (fjogeleit) [reply scripts/docker-build-entrypoint.sh:3]: @sonatype-lift ignore

**Comment by eddycharly** [comment]
The idea is to add a docker compose or just build the frontend in a container ?

**Comment by fjogeleit** [comment]
Just build the frontend in the container to remove the frontend dependencies for backend changes

**Comment by eddycharly** [comment]
Ok, not sure why we need docker compose though.

**Comment by fjogeleit** [comment]
Just simplification, don’t need it but it’s available anyway

**Comment by fjogeleit** [comment]
We can also remove it and use directly docker, as you like. But then we need to build the image first and then run it.

**Comment by eddycharly** [comment]
I was more or less thinking about something like this:
```
docker run --rm --entrypoint sh -v ${PWD}/frontend:/frontend -w /frontend node:20-alpine -c "npm install && npm run build"
```

**Comment by fjogeleit** [comment]
If this works also fine for me

**Comment by fjogeleit** [comment]
@eddycharly updated

**Review by eddycharly — APPROVED** [review]

**Pull request merged**